src/VisualStudio/Core/Def/Implementation/Workspace/VisualStudioDocumentNavigationService.cs

```diff
@@ -48,6 +48,13 @@ public bool CanNavigateToSpan(Workspace workspace, DocumentId documentId, TextSp
 
             var document = workspace.CurrentSolution.GetDocument(documentId);
```

Comment:
@heejaechang I have filed this work as https://github.com/dotnet/roslyn/issues/16367, since it's a larger change than this tactical crash-preventing fix. It's a good idea, though. :smile: Do you feel like *this* change should be gated on this additional work?